src/objects/DataImportBatch__c.object

```diff
@@ -86,6 +86,19 @@
         </displayedFields>
         <label>Batch Gift Entry Tab Columns</label>
     </fieldSets>
+    <fields>
```

Comment:
Why is this field necessary?  How is it used?

Reply:
This is definitely not necessary. Leftover from some explorations I was doing early on. Clearing this out shortly. Thanks!!